doc/_static/page_history.js

```
@@ -0,0 +1,105 @@
+
+        let link = `${home}${versionsArr[i]}/`
+        link += get_vdata(data.altnames, versionsArr[i], data.file, validlang);
+        if (versionsArr[i] == latest) result += '(' + link + ') ';
+        else result += link + ' ';
+    }
+    result += home + 'main/' + lang + '/' + data.file + '.html"> main </a> ';
+    result += home + 'dev/' + lang + '/' + data.file + '.html"> dev </a> ';
+
+    if (data.newat <= unsupportedArr[0]) {
+        result += '<br/><strong>' + title.v[1] + ':</strong>';
+        for (var i = 0; i < unsupportedArr.length; i++) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)